install_files/ansible-base/group_vars/securedrop.yml

```diff
@@ -58,19 +58,4 @@ grub_pax:
 
 grsec_package: securedrop-grsec
 
-### Used by the build-ossec-deb-pkg roles ###
-build_ossec_deb_pkg_dependencies:
-  - inotify-tools
-  - libssl-dev
-  - make
-  - tar
-  - unzip
-ossec_group: "ossec"
```

Comment:
ossec_group var is still required need to add it back. It is used to set file permissions for the ossec gpg keyring used by procmail to encrypt the alerts.